inc/render/class-masonry-variant.php

```diff
@@ -58,6 +58,14 @@ public function render_blocks( $block_content, $block ) {
 				true
 			);
 
+			wp_localize_script(
+				'otter-masonry-metadata',
+				'themeisleOtterMetadata',
+				array(
+					'useOldMacyContainer' => version_compare(get_bloginfo('version'),'5.9', '<'),
+				)
+			);
```

Comment:
@HardeepAsrani, I do not know why this is not working. Should `wp_localize_script` be used only in `class-main.php`?

Reply:
Nope, it can be used anywhere. The first argument should be the handle of the script. It should be `otter-masonry`